tests/miniconda_install_test.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -e
+
+DIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+# shellcheck source=../scripts/newinstall.sh
+. "${DIR}/../scripts/newinstall.sh"
+
+# "stub" out cmd to prevent curl from being run
+# shellcheck disable=SC2034
+cmd="echo"
```

Comment:
It is commented in {{newinstall.sh}}: https://github.com/lsst/lsst/blob/master/scripts/newinstall.sh#L933-L939 -- I'm not sure what else could be noted here that would be helpful.